scripts/frr/configs/commands/policy.json

```diff
@@ -1,14 +1,12 @@
 {
     "/policy/route/route-map/@enter": "!",
     "/policy/route/route-map/@element/rule/@element":"route-map {/../../tagnode/@text} {/action/@text} {/tagnode/@text}",
-   "/policy/route/route-map/@element/rule/@element/match/as-path":"match as-path {@text}",
+    "/policy/route/route-map/@element/rule/@element/match/as-path":"match as-path {@text}",
     "/policy/route/route-map/@element/rule/@element/match/interface":"match interface {@text}",
-    "/policy/route/route-map/@element/rule/@element/match/ip/address/standard-acl":"match ip address {@text}",
-    "/policy/route/route-map/@element/rule/@element/match/ip/address/extended-acl":"match ip address {@text}",
-    "/policy/route/route-map/@element/rule/@element/match/ip/address/prefix-lists":"match ip address prefix-list {@text}",
-    "/policy/route/route-map/@element/rule/@element/match/ip/nexthop/standard-acl":"match ip next-hop {@text}",
-    "/policy/route/route-map/@element/rule/@element/match/ip/nexthop/extended-acl":"match ip next-hop {@text}",
-    "/policy/route/route-map/@element/rule/@element/match/ip/nexthop/prefix-lists":"match ip next-hop prefix-list {@text}",
+    "/policy/route/route-map/@element/rule/@element/match/ip/address/access-list":"match ip address {@text}",
+    "/policy/route/route-map/@element/rule/@element/match/ip/address/prefix-list":"match ip address prefix-list {@text}",
+    "/policy/route/route-map/@element/rule/@element/match/ip/nexthop/access-list":"match ip next-hop {@text}",
+    "/policy/route/route-map/@element/rule/@element/match/ip/nexthop/prefix-list":"match ip next-hop prefix-list {@text}",
```

Comment:
These changes are based on the work by 'gbe0' and 'dewi-morgan' in the following pull request which is missing the DCO signed-off-by statement.

Reference: DAN-25